Quantum-Control-Applications-QuAM/Superconducting/calibrations/26_two_qubit_xeb.py

```diff
@@ -0,0 +1,62 @@
+import numpy as np
+from quam_libs.components import QuAM, TransmonPair
+from quam_libs.experiments.two_qubit_xeb import (
+    XEBConfig,
+    XEB,
+    backend as fake_backend,
+    QUAGate,
+)
+
+machine = QuAM.load()
+qubits = machine.active_qubits
+# Get the relevant QuAM components
+target_qubit_indices = [3, 4]  # Indices of the target qubits
+target_qubits = [qubits[i] for i in target_qubit_indices]
+target_qubit_pairs = [
+    qubit_pair
+    for qubit_pair in machine.active_qubit_pairs
+    if qubit_pair.qubit_control in target_qubits and qubit_pair.qubit_target in target_qubits
+]
+
+
+def cz_gate(qubit_pair: TransmonPair):
+    """
+    CZ gate macro
+    Here, we take inspiration from the macro implemented in QUA-Libs and adapt it to our needs using QuAM components.
+    """
+    pass
```

Comment:
Is this prompting the user to call the macro from QUA-libs? Or `quam_libs`? Could you make it clearer what the user should do or even add a commented call to the macro that now lives in `quam_libs`?

Reply:
Actually this is more to indicate the users to insert their own macros for implementing the gate. It is expected that the macro looks the same for all qubit pairs or that they cover special handling within the same macro for different qubit pairs (this is due to the fact that we're still missing the QUAM gate interface)

Reply:
I will change the doctoring here to explicitly request the user to insert their macro. I'm not sure which macro you are referring to.

Reply:
Yep that works!